api/package.json

```diff
@@ -38,7 +38,7 @@
     "cors": "^2.8.5",
     "csv-parse": "^4.4.6",
     "debug": "^4.1.1",
-    "detect-csv": "^1.1.0",
+    "csv-sniffer": "^0.1.1",
```

Comment:
What made you decide to switch library? (I'm interested because I'm also using "detect-csv" in the Bexio app)

Reply:
Turns out that `detect-css` does not detect the quote character

Reply:
OMG, just be careful. Without a closed set of allowed delimiters it produces weird results: https://runkit.com/embed/rb11tnz0n0he

Reply:
Haha! That's really weird, I wonder how they can get this result